samples/CMakeLists.txt

```diff
@@ -1,3 +1,5 @@
+cmake_minimum_required(VERSION 3.4.3)
+
 set(COMPUTECPP_SDK_INCLUDE ${CMAKE_CURRENT_SOURCE_DIR}/../external/computecpp-sdk/include)
 set(SYCLBLAS_INCLUDE ${CMAKE_CURRENT_SOURCE_DIR}/../include)
```

Comment:
If you are using the SYCL-Blas as a library you should use the target include directory and get the sycl-blas include pass from the library.